src/Pages/Home/Home.tsx

```diff
@@ -26,21 +27,18 @@ export const Home = () => {
   }, []);
```

Comment:
Here will be an error, if we leave the page, because we trying to call `setTestList` while component is unmounted